src/entry.h

```diff
@@ -109,6 +109,9 @@ void CacheEntryUnref(CacheEntry *self);
 CacheEntry *CacheEntryUnserialize(const gchar *value, GError **error);
 gchar *CacheEntrySerialize(CacheEntry *self);
 
+void CacheEntryHashSet(CacheEntry *self, CacheEntryAlgorithm alg,
+                  GBytes *hash, CacheEntryArgs *args);
```

Comment:
Are you sure you don't have a tab or something in here? It still looks wrong in Github.